packages/blocks/src/_common/utils/query.ts

```diff
@@ -822,7 +822,7 @@ export function getDropRectByPoint(
     const table = getDatabaseBlockTableElement(element);
     assertExists(table);
     let bounds = table.getBoundingClientRect();
-    if (model.isEmpty()) {
+    if (model.isEmpty.value) {
```

Comment:
from preact/signal doc, it looks like the recommended way of reading values is via `.peek()`

Reply:
Done